src/gym_electric_motor/physical_systems/electric_motors/SixPhase_PMSM.py

```diff
@@ -0,0 +1,169 @@
+    i_sy            A
+    i_salpha        A      Stator current in alpha direction
+    i_sbeta         A      Stator current in beta direction
+    i_sX            A
+    i_sY            A
+    i_sa1           A
+    i_sa2           A
+    i_sb1           A
+    i_sb2           A
+    i_sc1           A
+    i_sc2           A
+
+    =============== ====== =============================================
+    =============== ====== =============================================
+    Motor Voltages  Unit   Description
+    =============== ====== =============================================
+    u_sd            V      Direct axis voltage
+    u_sq            V      Quadrature axis voltage
+    u_sx            V
+    u_sx            V
+    =============== ====== =============================================
+
+    ======== ===========================================================
+    Limits / Nominal Value Dictionary Entries:
+    -------- -----------------------------------------------------------
+    Entry    Description
+    ======== ===========================================================
+    i        General current limit / nominal value
+    ======== ===========================================================
+        
+    """
+    I_SD_IDX = 0
+    I_SQ_IDX = 1
+    I_SX_IDX = 2
+    I_SY_IDX = 3
+    CURRENTS_IDX = [0, 1, 2, 3]
+    CURRENTS = ["i_sd", "i_sq", "i_sx", "i_sy"]
+    VOLTAGES = ["u_sd", "u_sq", "u_sx", "u_sx"]
+
+    @property
+    def motor_parameter(self):
+        # Docstring of superclass
+        return self._motor_parameter
+
+    @property
+    def initializer(self):
+        # Docstring of superclass
+        return self._initializer
+
+   #### Parameters taken from  https://ieeexplore.ieee.org/document/10372153
+    _default_motor_parameter = {"p": 5, "l_d": 125e-6, "l_q": 126e-6, "l_x": 39e-6, "l_y": 35e-6, "r_s": 64.3e-3, "psi_PM": 4.7e-3,}
+    #_default_limits = ?
+    #_default_nominal_values = ?
+    _default_initializer = {
+        "states": {"i_sd": 0.0, "i_sq": 0.0, "i_sx": 0.0, "i_sy": 0.0},
+        "interval": None,
+        "random_init": None,
+        "random_params": (None, None),
+    }
+
+    _model_constants = None
+
+    _initializer = None
+
+    def __init__(
+        self,
+        motor_parameter=None,
+        nominal_values=None,
+        limit_values=None,
+        motor_initializer=None,
+    ):
+        # Docstring of superclass
+        nominal_values = nominal_values or {}
+        limit_values = limit_values or {}
+        super().__init__(motor_parameter, nominal_values, limit_values, motor_initializer)
+        self._update_model()
+        self._update_limits()
+    
+    def _update_model(self):
+        """Updates the motor's model parameters with the motor parameters.
+
+        Called internally when the motor parameters are changed or the motor is initialized.
+        """
+        mp = self._motor_parameter
+        self._model_constants = np.array([
```

Comment:
It would be helpful to specify via comment which matrix column belongs to which regressor (please see other motor types for reference).